src/test/kotlin/com/personio/synthetics/builder/SyntheticBrowserTestBuilderTest.kt

```diff
@@ -0,0 +1,57 @@
+package com.personio.synthetics.builder
+
+import com.personio.synthetics.client.SyntheticsApiClient
+import com.personio.synthetics.config.getConfigFromFile
+import com.personio.synthetics.model.config.Location
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.mockito.Mockito
+import java.net.URL
+
+class SyntheticBrowserTestBuilderTest {
+    private lateinit var sut: SyntheticBrowserTestBuilder
+
+    @BeforeEach
+    fun prepareSut() {
+        val apiClientMock = Mockito.mock(SyntheticsApiClient::class.java)
+        sut = SyntheticBrowserTestBuilder(
+            "any_name",
+            getConfigFromFile("config-unit-test.yaml").defaults,
+            apiClientMock
+        )
+    }
+
+    @Test
+    fun `baseUrl sets base URL`() {
```

Comment:
would be nice if you could add more context in the test, 
`@Test
    fun 'baseUrl function sets the base url in the test config'`